addons/l10n_cl/views/account_move_view.xml

```diff
@@ -15,6 +15,31 @@
         </field>
     </record>
 
+    <record id="view_latam_form_inherit_l10n_cl" model="ir.ui.view">
+        <field name="name">account.move.latam.form.inherit.l10n.cl</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="l10n_latam_invoice_document.view_move_form"/>
+        <field name="arch" type="xml">
+            <field name="l10n_latam_document_number" position="attributes">
+                <attribute name="attrs">
+                    {'invisible': [
```

Comment:
invisible parameters seem correct, but the indentation of the first part can be improved.

Reply:
it's weird 🤔